### PR TITLE
Addon-docs: Add inline rendering for svelte

### PR DESCRIPTION
**Pull request by BlackFenix2**
Issue:

## What I did
i added inline rendering support for svelte components, i used a HOC.svelte file to mount the components passed into StoryFn() on the app. 
## How to test

run storybook on svelte-kitchen-sink to see the inline components.

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Comment by shilman** [comment]
@BlackFenix2 Hey, just got a chance to test this out and it's working well!

Can you clean up the PR a bit? Specifically:
- Remove all the auto-argtype stuff for a separate PR
- Go through the changes tab here and remove anything not related to inline rendering support?

Looking forward to merging this & releasing in 6.1-alpha!

**Comment by BlackFenix2** [comment]
ok, i deleted the unneeded files. everything in this pr is just for inline rendering support now.

File: addons/docs/src/frameworks/svelte/config.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { extractArgTypes } from './extractArgTypes';
+import { extractComponentDescription } from '../../lib/docgen';
+import { prepareForInline } from './prepareForInline';
+
+export const parameters = {
+  docs: {
+    inlineStories: true,
+    prepareForInline,
+    // extractArgTypes,
```
Comment (shilman) [inline addons/docs/src/frameworks/svelte/config.ts:9]:
```suggestion
```

File: addons/docs/src/frameworks/svelte/config.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    // extractComponentDescription,
```
Comment (shilman) [inline addons/docs/src/frameworks/svelte/config.ts:10]:
```suggestion
```

File: addons/docs/src/frameworks/svelte/config.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { extractArgTypes } from './extractArgTypes';
```
Comment (shilman) [inline addons/docs/src/frameworks/svelte/config.ts:1]:
```suggestion
```

File: addons/docs/src/frameworks/svelte/HOC.svelte [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<script>
+  import { onMount } from 'svelte';
+
+  let greeting = 'Svelte HOC';
```
Comment (shilman) [inline addons/docs/src/frameworks/svelte/HOC.svelte:4]:
```suggestion
```

File: addons/docs/src/frameworks/svelte/config.ts (hunk 2 above)
Comment (shilman) [inline addons/docs/src/frameworks/svelte/config.ts:10]:
```suggestion
```

**Review by shilman — CHANGES_REQUESTED** [review]
more cleaning

**Review by BlackFenix2 — COMMENTED** [review]
I finished cleaning up the requested files.

**Review by shilman — APPROVED** [review]
💥 boom! looking great.

**Comment by shilman** [comment]
I'll merge once we start 6.1, maybe later today or tomorrow.

**Comment by autoferrit** [comment]
Being new to storybook, is there a place that describes how to use this?

**Comment by shilman** [comment]
@autoferrit here's the introduction. unfortunately we're missing svelte code snippets

https://storybook.js.org/docs/svelte/writing-docs/introduction

@BlackFenix2 's inline rendering change should be applied automatically in 6.1 without any user intervention required

**Pull request merged**